### PR TITLE
[orc8r][gen] Automatically generate mockery mocks

### PR DESCRIPTION
**Pull request by hcgatewood**
## Summary

This PR contains two changes: (a) mockery-specific, and (b) side tangent where I cleaned up and standardized our code generation.

(a) This PR automatically generates the mockery mocks for blobstore pkg. Previously, these were updated by-hand when needed. Now, it happends automatically on build.py -g. I added this as an additional "make easy" for intern Andy, since he'll be updating the TransactionalBlobStore interface

(b) The way we're handling code generation (specifically: protos, swagger, and entgo) was a bit untidy. This just cleans things up a bit, though it's still somewhat imperfect

## Test Plan

- [x] Visual inspection
- [x] build.py -t

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by themarwhal — APPROVED** [review]

**Review by uri200 — APPROVED** [review]

File: orc8r/cloud/Makefile [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
-##########################################
-## Some Useful Directory/File Variables ##
-##########################################
-# Allow MAGMA_ROOT and PLUGIN_DIR to be set from environment to support travis
+################
+## Essentials ##
+################
 ifndef MAGMA_ROOT
 MAGMA_ROOT := /home/$(USER)/magma
+$(warning MAGMA_ROOT is undefined, using default: $(MAGMA_ROOT))
```
Comment (emakeev) [inline orc8r/cloud/Makefile:8]: 👍

File: orc8r/cloud/Makefile [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
 
+ifndef MAGMA_MODULES
+MAGMA_MODULES := \
+	$(MAGMA_ROOT)/orc8r \
+	$(MAGMA_ROOT)/lte \
+	$(MAGMA_ROOT)/feg \
+	$(MAGMA_ROOT)/cwf \
+	$(MAGMA_ROOT)/wifi \
+	$(MAGMA_ROOT)/fbinternal
+$(warning MAGMA_MODULES is undefined, using default: $(MAGMA_MODULES))
```
Comment (emakeev) [inline orc8r/cloud/Makefile:20]: 👍

**Review by emakeev — APPROVED** [review]
LGTM

**Comment by hcgatewood** [comment]
[wip]

**Review by mpgermano — APPROVED** [review]

**Pull request merged**